### DISCUSSION TITLE
Any Possibility there will be a tool to deal with the Direct Publisher sunsetting on roku/json and deal with brightscript and scenegraph?

### DISCUSSION THREAD

**Discussion by peaktv** [rrirower/Channel-Feed-Manager#141] [General]
So The channel feed manager for Roku is an awesome tool and was a great advantage in dealing with our json file creation and getting and keeping our channel running. We/I am dealing with this new sdk way of updating our channel and as a programmer that has skills that lapsed decades ago I am ready to give up on this new roku environment. In turn, I am hoping the author has something in the works to deal with this development environment. Maybe im missing something but the best Ive gotten so far is a side load with the hello world sample to load. . . our site is very simple but Ive grown weary trying to migrate. . .  . Im hoping there's hope for a new/updated tool . . .

**Comment by rrirower** [comment]
Have you tried the DP2SceneGraphWizard tool available on this site? It's can generate a Roku channel for you.

**Comment by peaktv** [comment]
I havent. . ..  frankly the only person that has had any true knowledge on this that has guided me to success is you. My plan at this point is to use your channel feed manager and run it thru that sdk converter ive seen and try to make that work. . . . I will give it a shot. I truly appreciate your efforts - you've made a great tool and I love using it. . . have a great day!


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Comment by peaktv** [comment]
so I just downloaded it and I think thats the answer. I realllly appreciate you . . .


Sent from Outlook<http://aka.ms/weboutlook>

________________________________
